build.sbt

```diff
@@ -8,11 +8,17 @@ scalaVersion in ThisBuild := "2.11.8"
 
 version := "wk"
 
+scalacOptions in ThisBuild ++= Seq(
+  "-target:jvm-1.8",
+  "-feature",
+  "-deprecation",
```

Comment:
I guess `-feature` and `-deprecation` increase output, but how exactly?

Reply:
Feature warnings are not warnings generated by the scala compiler but warnings defined by the libraries. without this compiler flag, the output would show a single warning with the text »there were 22 feature warnings«, now it shows their full output. Same with deprecation warnings.